recipes/biorbd-optim/yum_requirements.txt

```diff
@@ -0,0 +1,2 @@
+xorg-x11-server-Xvfb
+libXtst
```

Comment:
These should have CDTs associated with them

Reply:
Thanks for the review!
I don't know what you mean, do you have an example that I can use?

Reply:
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#host